package.json

```diff
@@ -67,7 +67,7 @@
   },
   "dependencies": {
     "@konveyor/lib-ui": "^1.4.0",
-    "@patternfly/react-core": "^4.40.4",
+    "@patternfly/react-core": "4.47.1",
```

Comment:
Should be "@patternfly/react-core": "^4.47.0" 
Looks like 4.47.0 is the latest version https://www.npmjs.com/package/@patternfly/react-core.
Also, we need to include the caret as we do for the other versions. This will allow for small version bumps & keep things consistent w respect to semantic versioning. Thx

Reply:
4.47.0 is latest published version on npm and it works too with Treeview.
Adding the caret too. 
Thanks